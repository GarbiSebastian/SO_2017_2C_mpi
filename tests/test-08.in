member estas
member palabras
member no
member deben
member aparecer
load tests/archivos/palabras-1
member todas
member las
member palabras
member de
member este
member archivo
member deben
member aparecer
member con
member member
member sin
member embargo
member estas
member no
quit
