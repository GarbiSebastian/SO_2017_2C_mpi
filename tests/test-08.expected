Comandos disponibles:
  load <arch_1> <arch_2> ... <arch_n>
  addAndInc <string>
  member <string>
  maximum
  q|quit
> El string <estas> no está
> El string <palabras> no está
> El string <no> no está
> El string <deben> no está
> El string <aparecer> no está
> La listá esta procesada
> El string <todas> está
> El string <las> está
> El string <palabras> está
> El string <de> está
> El string <este> está
> El string <archivo> está
> El string <deben> está
> El string <aparecer> está
> El string <con> está
> El string <member> está
> El string <sin> no está
> El string <embargo> no está
> El string <estas> no está
> El string <no> no está
> Finalizamos ejecución
